app/scripts/views/complete_sign_up.js

```diff
@@ -94,12 +96,14 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
                   self.navigate('settings', {
                     success: t('Account verified successfully')
                   });
+                  self._notifications.broadcast(self._notifications.EVENTS.SIGNED_IN, account.toJSON());
```

Comment:
mmm..  I never answered, and good question. I think for our own sanity, the other tab should be the one to make that decision, even if the current tab does not go to `settings`. If the other tab's broker has support for the `handleSignedInNotification` capability, it shouldn't make much difference to it where the verification tab goes.